tests/files.fish

```diff
@@ -5,7 +5,9 @@ set temp (command mktemp -d)
 builtin cd $temp
 
 @test "a directory" -d $temp
+@test "a non-existent directory" ! -d $temp.fake
```

Comment:
As you mentioned earlier, Fishtape can already evaluate `test` expressions that contain a `!`. It would be great if we could test this in `tap.fish` instead.